options.go

```diff
@@ -81,6 +83,15 @@ func Context(ctx context.Context) Option {
 	}
 }
 
+// SignalHandler toggles automatic installation of the signal handler.
+// Users of this feature to disable the signal handler, should control liveness
+// of the service through the context.
+func SignalHandler(b bool) Option {
```

Comment:
HandleSignal might make more sense since the option will be called as

```
micro.NewService(
    micro.HandleSignal(bool),
)
```

Reply:
sg